java/shims/src/java/org/apache/orc/impl/ZeroCopyShims.java

```diff
@@ -51,6 +52,7 @@ private static final class ZeroCopyAdapter implements HadoopShims.ZeroCopyReader
         .noneOf(ReadOption.class);
     private static final EnumSet<ReadOption> NO_CHECK_SUM = EnumSet
         .of(ReadOption.SKIP_CHECKSUMS);
+    private final IdentityHashMap<ByteBuffer, Object> readBuffers = new IdentityHashMap<>(0);
```

Comment:
Could you add some descriptions because we use `IdentityHashMap`?

Reply:
Added